extra/catena-message-model-4905-sapflow-ttn-v2.js

```diff
@@ -95,36 +95,32 @@ function Decoder(bytes, port) {
             }
 
             var LiterPerGallon = 3.78541178;
-            var CfPerGallonPropane = 35.97;
-            var CfPerLiterPropane = CfPerGallonPropane / LiterPerGallon;
+            var CfPerGallonSap = 35.97;
+            var CfPerLiterSap = CfPerGallonSap / LiterPerGallon;
             // from https://www.eia.gov/energyexplained/units-and-calculators/british-thermal-units.php
-            var BTUperGallonPropane = 91452;
-            var BTUperCfPropane = BTUperGallonPropane / CfPerGallonPropane;
+            var BTUperGallonSap = 91452;
+            var BTUperCfSap = BTUperGallonSap / CfPerGallonSap;
         
             if (flags & 0x20) {
                 // fuel, spare
                 var pulseIn = (bytes[i] << 8) + bytes[i + 1];
-                i += 2;
-                var pulseOut = (bytes[i] << 8) + bytes[i + 1];
-                i += 2;
-                decoded.propaneUsedCF = pulseIn;
-                decoded.propaneUsedLiters = pulseIn / CfPerLiterPropane;
-                decoded.propaneUsedBTU = pulseIn * BTUperCfPropane;
+                i += 4;
+                decoded.sapUsedCF = pulseIn;
+                decoded.sapUsedLiters = pulseIn / CfPerLiterSap;
+                decoded.sapUsedBTU = pulseIn * BTUperCfSap;
```

Comment:
See above. Get rid of comments about fuel, etc.

Basically: don't just hack this code, go back to first principles and implement so that it produces relevant data about sap, and omits other irrelevant stuff.